### PR TITLE
Sketcher Handling Plurals for DoF string

### PR DESCRIPTION
**Pull request by Kuzma30**
Thank you for creating a pull request to contribute to FreeCAD! To ease integration, we ask you to conform to the following items. Pull requests which don't satisfy all the items below might be rejected. If you are in doubt with any of the items below, don't hesitate to ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10)!

- [ ]  Your pull request is confined strictly to a single module. That is, all the files changed by your pull request are either in `App`, `Base`, `Gui` or one of the `Mod` subfolders. If you need to make changes in several locations, make several pull requests and wait for the first one to be merged before submitting the next ones
- [ ]  In case your pull request does more than just fixing small bugs, make sure you discussed your ideas with other developers on the FreeCAD forum
- [ ]  Your branch is [rebased](https://git-scm.com/docs/git-rebase) on latest master `git pull --rebase upstream master`
- [ ]  All FreeCAD unit tests are confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ]  All commit messages are [well-written](https://chris.beams.io/posts/git-commit/) ex: `Fixes typo in Draft Move command text`
- [ ]  Your pull request is well written and has a good description, and its title starts with the module name, ex: `Draft: Fixed typos`
- [ ]  Commit messages include `issue #<id>` or `fixes #<id>` where `<id>` is the issue ID number from our [Issues database](https://github.com/FreeCAD/FreeCAD/issues) in case a particular commit solves or is related to an existing issue. Ex: `Draft: fix typos - fixes #4805`

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.20 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=56135).

---

### CODE REVIEW

**Comment by adrianinsaval** [comment]
isn't Degrees of Freedom DoF already plural? Also IIRC this string was shortened to just DoF on purpose.

File: src/Mod/Sketcher/Gui/ViewProviderSketch.cpp [hunk 1]
```diff
@@ -2910,8 +2910,8 @@ void ViewProviderSketch::UpdateSolverInformation()
     } else if (dofs > 0) {
         signalSetUp(QString::fromUtf8("under_constrained"),
             tr("Under constrained:"),
-            QString::fromUtf8("#dofs"),
-            QString::fromUtf8("%1 %2").arg(dofs).arg(tr("DoF")));
+            QObject::tr("Number of DoFs"),
```
[the remark this answers is not in the document]
Comment (Kuzma30) [reply src/Mod/Sketcher/Gui/ViewProviderSketch.cpp:2913]: @chennes I understand. Can you make commit for add plural and close this PR? (I not near computer now)

**Comment by chennes** [comment]
Sure: c9cffae11d25f0d1f1bf3c8533e757d9a65ac5d9

**Comment by chennes** [comment]
> isn't Degrees of Freedom (DoF) already plural?

In English, I'd agree. But the issue is that with other languages pluralization works differently, and the different language teams decide on their own how to handle it. I think we probably should go into the English translation and standardize on "DoF" for all values of `n`, but this change will allow us to do that.

**Comment by chennes** [comment]
Technically the source text is simply the default: we can provide "translations" in English, and it's expected that you do so in pluralization cases like this one.

**Pull request closed without merging**